src/Compilers/Core/Portable/Symbols/NullabilityInfo.cs

```diff
@@ -6,11 +6,21 @@
 
 namespace Microsoft.CodeAnalysis
 {
-    // PROTOTYPE(nullable-api): Doc Comment
     [DebuggerDisplay("{GetDebuggerDisplay(), nq}")]
     public readonly struct NullabilityInfo : IEquatable<NullabilityInfo>
     {
+        /// <summary>
+        /// The nullable annotation of the expression represented by the syntax node. This represents
+        /// the types of expressions that can be assigned to this expression, if this expression
+        /// can be used as an lvalue.
```

Comment:
Perhaps "the _nullability_ of ... _when_ this expression _is_ used as an lvalue".

Reply:
I'll correct the first part, but the second part was deliberate wording on my part.

---
In reply to: [276386403](https://github.com/dotnet/roslyn/pull/35047#discussion_r276386403) [](ancestors = 276386403)